acf/index.php

```diff
@@ -0,0 +1,3 @@
+<?php
```

Comment:
This is the recommendation in the ACF docs, seems sensible to follow it.

I agree they should set-up their server properly, but they may not.